content/applications/general/users/access_rights.rst

```diff
@@ -2,66 +2,184 @@
 Access Rights
 =============
 
-Activate the :ref:`developer mode <developer-mode>`, then go to :menuselection:`Settings --> Users &
-Companies --> Groups`.
+*Access rights* are permissions that determine the content and applications users can access and
+edit. In Odoo, these permissions can be set for individual users or for groups of users. Limiting
+permissions to only those who need them ensures that users do not modify or delete anything they
+should not have access to.
 
-Groups
-======
+**Only** an *administrator* can change access rights.
 
-| When choosing the groups the user can have access under
-  :ref:`Access Rights <users/add-individual>`, details of the rules and inheritances of that group
-  are not shown, so this is when the menu *Groups* comes along. *Groups* are created to define rules
-  to models within an application.
-| Under *Users*, have a list of the current ones. The ones with administrative rights are shown
-  in black.
+.. danger::
+   Making changes to access rights can have a detrimental impact on the database. This includes
+   *impotent admin*, which means that no user in the database can make changes to the access rights.
+   For this reason, Odoo recommends contacting an Odoo Business Analyst, or our Support Team, before
+   making changes.
 
-.. image:: access_rights/groups-users.png
-   :align: center
-   :alt: View of a group’s form emphasizing the tab users in Odoo
+.. tip::
+   A user **must** have the specific *Administration* access rights set on their user profile, in
+   order to make changes on another user's settings for access rights.
+
+   To access this setting, navigate to :menuselection:`Settings app --> Manage users --> Select a
+   User --> Access Rights tab --> Administration section --> Administration field`.
+
+   Once at the setting, an already existing administrator **must** change the setting in the
+   :guilabel:`Administration` field to :guilabel:`Access Rights`.
+
+   Once complete, click :guilabel:`Save` to save the changes, and implement the user as an
+   administrator.
+
+Users
+=====
 
-*Inherited* means that users added to this application group are automatically added to the
-following ones. In the example below, users who have access to the group *Administrator* of *Sales*
-also have access to *Website/Restricted Editor* and *Sales/User: All Documents*.
+The :ref:`Access Rights <users/add-individual>` for individual users are set when the user is added
+to the database, but they can be adjusted at any point in the user's profile.
 
-.. image:: access_rights/groups-inherited.png
+To make changes to a user's rights, click on the desired user to edit their profile.
+
+.. image:: access_rights/navigate-to-users-menu.png
    :align: center
-   :height: 330
-   :alt: View of a group’s form emphasizing the tab inherited in Odoo
+   :alt: In developer mode, go to the settings page and click Users and Companies, then Users.
```

Comment:
While a descriptive alt text is greatly appreciated, [it should not repeat the previous sentence instruction](https://www.odoo.com/documentation/17.0/contributing/documentation/content_guidelines.html#alt-tags), and it should just describe the image :)
```suggestion
   :alt: Users menu in the Users & Companies section of the Settings app of Odoo.
```